qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/dynamic/Utils.java

```diff
@@ -63,4 +65,17 @@ public static void assertBrokerDoesNotHavePartition(
             b -> b.getPartitions().stream().noneMatch(p -> p.getId() == partitionId),
             "Broker %d does not have partition %d".formatted(brokerId, partitionId));
   }
+
+  public static void assertClusterBecomesHealthy(
```

Comment:
:wrench: You can move this to `TestCluster` because this is reusable in other tests.